lib/openc_bot/tasks.rb

```diff
@@ -354,7 +354,7 @@ def get_bot_name
   end
 
   def only_process_running(task_name)
-    pid_path = File.join(PID_DIR, 'pids', task_name)
```

Comment:
@sebbacon Is it deliberate to make the pid_path have 'pids' in twice, i.e. /oc/pids/pids/[task_name] ?